plugins/main/public/components/overview/mitre/intelligence/intelligence.tsx

```diff
@@ -93,19 +95,26 @@ export const ModuleMitreAttackIntelligence = compose(
   );
 
   useEffect(() => {
-    const urlParams = new URLSearchParams(location.href);
-    const redirectTab = urlParams.get('tabRedirect');
-    if (redirectTab) {
-      setSelectedResource(redirectTab);
+    if (!paramTabRedirect) {
+      NavigationService.getInstance().updateAndNavigateSearchParams({
+        tabRedirect: selectedResource,
+      });
     }
   }, []);
```

Comment:
Shouldn't it be in the useEffect paramTabRedirect dependency listing? If not, if the component is rendered and the query param paramTabRedirect is removed from the URL, the first time it does not change, then upon reloading the expected effect occurs.

Reply:
Solved [bc8a973](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6882/commits/bc8a973fce188227916617db01ab1ad787114712)